src/main/java/slimeknights/tconstruct/common/TinkerTags.java

```diff
@@ -72,6 +82,8 @@
     public static final IOptionalNamedTag<Item> BLUE_SLIMEBALL = forgeTag("slimeball/blue");
     public static final IOptionalNamedTag<Item> PURPLE_SLIMEBALL = forgeTag("slimeball/purple");
     public static final IOptionalNamedTag<Item> BLOOD_SLIMEBALL = forgeTag("slimeball/blood");
+    public static final IOptionalNamedTag<Item> RAINBOW_SLIMEBALL = forgeTag("slimeball/rainbow");
+    public static final IOptionalNamedTag<Item> GOLD_SLIMEBALL = forgeTag("slimeball/gold");
```

Comment:
Might be easier to leave gold slime out for now. We can add it when we add the mob. Rather have it a complete feature than a future feature until we finish